global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src", "test/WebSites", "samples"]
+    "projects": ["src", "test/WebSites", "samples"],
+    "sdk": {
+      "version": "1.0.0-preview2-003154"
```

Comment:
It should be good for now. @JunTaoLuo, we'll need to update the CLI to the version that's shipping in the release (if any). Could you file a tracking item for this?